roles/ci_gen_kustomize_values/templates/common/network-values/values.yaml.j2

```diff
@@ -78,8 +78,14 @@ data:
       {
         "cniVersion": "0.3.1",
         "name": "{{ network.network_name }}",
+{%  if network.network_name == "octavia" %}
+        "type": "bridge",
+{%  else %}
         "type": "macvlan",
-{%  if network.vlan_id is defined%}
+{%  endif %}
+{%  if network.network_name == "octavia" %}
+        "bridge": "octbr",
```

Comment:
why don't you put this line in the first `if` so we can avoid a second iteration?